packages/composites/anchor/src/endpoint/price/beth.ts

```diff
@@ -43,24 +43,32 @@ const getStETHExchangeRate = async (
     curvePoolAbi,
     provider,
   )
-  const result = await stEthPoolContract.get_dy(1, 0, ethers.BigNumber.from(10).pow(18))
+  const baseTokenIndex = 1 // stETH has a token index of 1 in the curve pool
+  const quoteTokenIndex = 0 // ETH has a token index of 0 in the curve pool
+
+  // Gets how much ETH the pool will return if 1 stETH is traded.  The pool uses 18dp so 1 = 10**18.
+  const result = await stEthPoolContract.get_dy(
+    baseTokenIndex,
+    quoteTokenIndex,
+    ethers.BigNumber.from(10).pow(18),
+  )
   throwErrorForInvalidResult(
     jobRunID,
     result,
-    `stETH/ETH Exchange Rate from Curve Pool address ${config.stEthPoolContractAddress}`,
+    `ETH/stETH Exchange Rate from Curve Pool address ${config.stEthPoolContractAddress}`,
   )
   return result
 }
 
 /**
- * getStEthBEthExchangeRate returns a promise for the value of stETH/bETH
- * from the stETH/bETH Anchor Vault contract
+ * getBEthStETHExchangeRate returns a promise for the value of bEth/StETH
+ * from the bEth/StETH Anchor Vault contract
  * @param jobRunID string
  * @param config Config
  * @param provider ethers.providers.JsonRpcProvider
  * @returns Promise<ethers.BigNumber>
  */
-const getStEthBEthExchangeRate = async (
```

Comment:
old: stETH/bETH exchange rate
new: bEth/stETH exchange rate

Is there a reason for changing the name of the function if we're not changing the calculation? If we're only changing the name, then that means we're not truly making the correct adjustment and the math is incorrect.

Reply:
actually looking at this again `bETH/stETH` is correct.  The previous version is wrong.